pom.xml

```diff
@@ -324,7 +324,7 @@
         <dependency>
             <groupId>org.apache.jackrabbit.vault</groupId>
             <artifactId>org.apache.jackrabbit.vault</artifactId>
-            <version>3.4.0</version>
+            <version>3.6.1-T20220805092633-0b8a2641</version>
```

Comment:
Could you depend on the latest officially released version, `3.6.0` ?

Reply:
In that version we don't have the `LEGACY` conflict policy.